Unity/2019.1/Events with Caliper/Assets/Plugins/caliper-net/Release/ImsGlobal.Caliper.dll.config

```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <runtime>
+    <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
+      <dependentAssembly>
+        <assemblyIdentity name="Newtonsoft.Json" publicKeyToken="30ad4fe6b2a6aeed" culture="neutral" />
+        <bindingRedirect oldVersion="0.0.0.0-7.0.0.0" newVersion="7.0.0.0" />
```

Comment:
What is the purpose of this configuration?

Reply:
This was generated by Visual Studio on Windows when I built the Caliper-net dll.